### PR TITLE
Remove writeHaulerManifest, its template, and its tests

### PR DESCRIPTION
**Pull request by dbw7**

### CODE REVIEW

**Review by atanasdinov — APPROVED** [review]

**Review by jdob — APPROVED** [review]

File: pkg/combustion/registry.go [hunk 1]
```diff
@@ -232,6 +232,8 @@ func configureEmbeddedArtifactRegistry(ctx *image.Context) (bool, error) {
 		}
 	}
 
+	zap.S().Infof("Preparing to add the following images to the embedded artifact registry:\n%s", images)
```
Comment (atanasdinov) [inline pkg/combustion/registry.go:235]: I'd move this log just before `populateRegistry()` or even within it and reword this to `Adding the following images to the embedded artifact registry:`.

**Review by atanasdinov — APPROVED** [review]
